src/lib/app-globals.ts

```diff
@@ -12,3 +12,17 @@ export function isMac() {
 export function isWindows() {
 	return getAppGlobals().platform === 'win32';
 }
+
+export function getPlatformName() {
+	const platform = process ? process.platform : getAppGlobals().platform;
```

Comment:
For now, this function is only used in the main process (which has access to `process` object and node). However, I also added support for invocations from the renderer process via the app globals, in case we need it in the future.

Reply:
Good idea. 👍